### ISSUE TITLE
ERT Selection is not working correctly

### ISSUE THREAD

**Issue by Taaketa** [ParadiseSS13/Paradise#5548]
**Problem Description**:
What is the problem?

I've seen this 3 times now. When an ERT is called by the station the ERT numbers are always very low. Yesterday an ERT squad consisted of 3 people. Today an ERT squad consisted of 2 people.

**What did you expect to happen**: 
Why do you think this is a bug?

I understand the mechanics of ERT selection are. Observers first. Dead Players second. It's my belief that afk Observers are causing issues with selection. That is not obvious the ERT team in place that there is a delay to selection of dead players and observers.

The underlying issue is several times now I've seen plenty of ghosts (observers and dead players) and only 2 or 3 ERT are provided even though there is a large selection of ghosts to take from.

**What happened instead**:
How is what happened different from what you expected?

ERT gathering quicker when selection comes up. Observers who are AFK maybe delaying the selection as they have the "DO you wish to be a member of ERT?" message on screen but have not yet selected an outcome.

**Comment by FreeStylaLT** [comment]
@Krausus

**Comment by Krausus** [comment]
> It's my belief that afk Observers are causing issues with selection
> Observers who are AFK maybe delaying the selection as they have the "DO you wish to be a member of ERT?" message on screen but have not yet selected an outcome.

There is always exactly 60 seconds between asking players if they want to join the ERT and actually spawning the selected members, regardless of how many players do or do not select an option. Players who do not answer, whether AFK or otherwise, are simply ignored.

> Observers first. Dead Players second.

The second group is actually players who have bailed from the round very early. Players who die and ghost normally have the same priority as direct observers. And it doesn't matter for selection, because everyone is picked simultaneously.

> The underlying issue is several times now I've seen plenty of ghosts (observers and dead players) and only 2 or 3 ERT are provided even though there is a large selection of ghosts to take from.

Unfortunately, that's how many ghosts signed up. Many situations that would call for an ERT result in many people choosing to simply observe things unfold, or stay a ghost and hope they'll be brought into the round as a mid-round antagonist (see: xenos). The only way to change this would be to force people to join the ERT (we aren't likely to be doing that) or find some way to incentivize joining it.

**Comment by Taaketa** [comment]
The reason why I raise it though. Is because for 2 rounds I was a dead player. ERT got called and only 2 appeared. I was never offered once to join the ERT.

**Comment by Krausus** [comment]
The only thing that should actually prevent you from joining the ERT is having AntagHUD turned on. Did you have it enabled on the occasions you didn't get a choice?

**Comment by Taaketa** [comment]
Nope. On both occasions it was a cult round and a traitor/ling round which I discovered through the chat log window without the need of the antag hud. First round I was an IPC in Robotics. It was Blob and only 2 ERT spawned. I had died fighting the blob and ghosted from my corpse.

Second round. I had died due to an alien taking my body, infecting me and then getting fed up of my constant antics of escaping, so they killed me. I ghosted again.

On both occasions I was a ghost and the ERT prompt did not appear. But there were ERT members but only 2.
